### PR TITLE
Adds a chat box + bwoink sound to the Admin Message Antagonist Team button

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
As admins can now more easily make teams, sometimes info needs to be made known. 
When this is your message it can very easily be lost in a fight. A message box + the ticket opening sound should add to help make known that an admin just tried to message everyone.
image

## Why It's Good For The Game
Admin messages shouldn't be easily missable

## Testing
Made myself a member of a team, messaged myself
image

## Changelog
:cl:
add: [ADMIN] Messaging a team now plays a ticket open soundeffect + chat box for the team members
/:cl:

### CODE REVIEW

**Comment by Contrabang** [comment]
IMO, the box, and having a sound is good, but we need to stop putting the bwoink sound everywhere, and instead use a different alert sound. I think AA also said a similar thing for the server announcements.

**Comment by DGamerL** [comment]
> IMO, the box, and having a sound is good, but we need to stop putting the bwoink sound everywhere, and instead use a different alert sound. I think AA also said a similar thing for the server announcements.

I missed AA saying that, got any recommendations for a sound?

File: code/modules/antagonists/_common/antag_team.dm [hunk 1]
```diff
@@ -168,7 +168,8 @@ GLOBAL_LIST_EMPTY(antagonist_teams)
 		return
 
 	for(var/datum/mind/M as anything in members)
-		to_chat(M.current, "<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>")
+		SEND_SOUND(M.current, sound('sound/effects/adminhelp.ogg', 0, 1, 0, 25))
+		to_chat(M.current, chat_box_red("<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>"))
```
Comment (Burzah) [inline code/modules/antagonists/_common/antag_team.dm:172]:
```suggestion
		to_chat(M.current, chat_box_red("<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span></font><span class='notice'>[message]</span>"))
```
I know that the second span class for the message will set a font color separate from the font tag, but font should still be closed as it is a container tag.

**Comment by Contrabang** [comment]
> > IMO, the box, and having a sound is good, but we need to stop putting the bwoink sound everywhere, and instead use a different alert sound. I think AA also said a similar thing for the server announcements.
> 
> I missed AA saying that, got any recommendations for a sound?

Perhaps 'sound/effects/adminticketopen.ogg'?

**Comment by DGamerL** [comment]
> > > IMO, the box, and having a sound is good, but we need to stop putting the bwoink sound everywhere, and instead use a different alert sound. I think AA also said a similar thing for the server announcements.
> > 
> > 
> > I missed AA saying that, got any recommendations for a sound?
> 
> Perhaps 'sound/effects/adminticketopen.ogg'?

LGTM, will add it

File: code/modules/antagonists/_common/antag_team.dm [hunk 2]
```diff
@@ -168,7 +168,8 @@ GLOBAL_LIST_EMPTY(antagonist_teams)
 		return
 
 	for(var/datum/mind/M as anything in members)
-		to_chat(M.current, "<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>")
+		SEND_SOUND(M.current, sound('sound/effects/adminticketopen.ogg'))
+		to_chat(M.current, chat_box_red("<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span></font><span class='notice'>[message]</span>"))
```
Comment (Contrabang) [inline code/modules/antagonists/_common/antag_team.dm:172]: Can you make this into a var before the looping, so it doesnt have to so many string concatenations? Possible the same for the sound() call.

Comment (lewcc) [inline code/modules/antagonists/_common/antag_team.dm:172]: It probably couldn't hurt to include the team name, either shown to the players and/or (especially if the team name is used more internally than not) in the admin logs

Comment (DGamerL) [reply code/modules/antagonists/_common/antag_team.dm:172]: Included team name in the next commit!

File: code/modules/antagonists/_common/antag_team.dm [hunk 3]
```diff
@@ -167,8 +167,11 @@ GLOBAL_LIST_EMPTY(antagonist_teams)
 	if(!message)
 		return
 
+	var/team_message = chat_box_red("<font color='#d6000b'><span class='bold'>Admin '[name]' Team Message ([user.key]): </span></font><span class='notice'>[message]</span>")
+	var/team_alert_sound = sound('sound/effects/adminticketopen.ogg')
 	for(var/datum/mind/M as anything in members)
-		to_chat(M.current, "<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>")
+		SEND_SOUND(M.current, team_alert_sound)
+		to_chat(M.current, "[team_message]")
```
Comment (Contrabang) [inline code/modules/antagonists/_common/antag_team.dm:174]: This is already a text message, isnt it? No need to put it in a string

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]
address contrabangs comment and then this will be good to go

File: code/modules/antagonists/_common/antag_team.dm (hunk 3 above)
Comment (DGamerL) [reply code/modules/antagonists/_common/antag_team.dm:174]: I blame Sean ... somehow

File: code/modules/antagonists/_common/antag_team.dm [hunk 4]
```diff
@@ -167,8 +167,11 @@ GLOBAL_LIST_EMPTY(antagonist_teams)
 	if(!message)
 		return
 
+	var/team_message = chat_box_red("<font color='#d6000b'><span class='bold'>Admin '[name]' Team Message ([user.key]): </span></font><span class='notice'>[message]</span>")
+	var/team_alert_sound = sound('sound/effects/adminticketopen.ogg')
 	for(var/datum/mind/M as anything in members)
-		to_chat(M.current, "<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>")
+		SEND_SOUND(M.current, team_alert_sound)
```
Comment (lewcc) [inline code/modules/antagonists/_common/antag_team.dm:173]: Probably best to add a check here while you're at it
```suggestion
		if(QDELETED(M.current))
			continue
		SEND_SOUND(M.current, team_alert_sound)
```

**Review by lewcc — APPROVED** [review]

**Pull request merged**